src/index.js

```diff
@@ -52,8 +52,19 @@ listItemsContainer.addEventListener('keypress', (e) => {
       }
       const num = e.target.getAttribute('data-para');
       Tasks[num].description = e.target.textContent;
-      window.location.reload();
+      // window.location.reload();
     }
   }
   return localStorage.setItem('list', JSON.stringify(Tasks));
-});
\ No newline at end of file
+});
+
+listItemsContainer.addEventListener('click', (event) => {
+  check(event);
+});
+
+document.addEventListener('change', (event) => {
+  UpdateCheck(event);
+});
+
+const deleteBtn = document.getElementById('btn-li');
+deleteBtn.addEventListener('click', DeleteAll);
```

Comment:
Adding and Removing todos are part of this milestone, I get an error when adding a todo 
<img width="1144" alt="image" src="https://user-images.githubusercontent.com/67957533/229079231-bf781053-6e0b-4e9f-80b5-133f52738600.png">

- [ ] Kindly work this issue